nginx.conf

```diff
@@ -36,7 +59,6 @@ http {
    }
 
    location @index {
-     add_header Cache-Control no-cache;
```

Comment:
@ozamani9 Why did you remove this?

Reply:
it was added in the header section instead.  so its like i moved it, not removed it.. does that make sense

![image](https://user-images.githubusercontent.com/20173559/150870229-0ed50f30-7b55-4266-95f1-13785ba2f9fc.png)

Reply:
Yes, perfect.